src/test/java/com/smalaca/messagesender/service/MessageCrudTest.java

```diff
@@ -56,4 +57,50 @@ public void shouldNotCreateNewMessage() {
 
         Assert.assertFalse(response.isSuccess());
     }
+
+    @Test
+    public void shouldDeletePreviouslyCreatedNewMessage() {
+
+        String subject = "subject";
+        String body = "body";
+        String from = "from";
+        String to = "to";
+
+        MessageDto messageDto = new MessageDto();
+        messageDto.setBody(body);
+        messageDto.setSubject(subject);
+        messageDto.setFrom(from);
+        messageDto.setTo(to);
+
+        messageCrud.createNew(messageDto);
```

Comment:
to jest część Waszych założeń, więc:
1. Nowa linia powinna pojawić się nie w 74 linia, a po wywołaniu tego, co rzeczywiście testujecie (linia 78)